src/dsp_tools/commands/validate_data/reformat_validaton_result.py

```diff
@@ -361,6 +356,24 @@ def _reformat_one_validation_result(validation_result: ValidationResult) -> Inpu
             raise BaseError(f"An unknown violation result was found: {validation_result.__class__.__name__}")
 
 
+def _reformat_min_cardinality_validation_result(validation_result: ResultMinCardinalityViolation) -> InputProblem:
+    iris = _reformat_main_iris(validation_result)
+    file_value_properties = ["shapes:hasGenericFileValue"]
```

Comment:
I will change the identification of the problem to looking at the source shape. Because when a wrong file value is used, we will get two violations, one min card and one closed shapes.
But I may refactor this separately, I will have to look at how big the diff is. But for now this works.